storage/storage.go

```diff
@@ -1137,24 +1137,21 @@ func (o *ObjectAttrs) toRawObject(bucket string) *raw.Object {
 
 // toProtoObject copies the editable attributes from o to the proto library's Object type.
 func (o *ObjectAttrs) toProtoObject(b string) *storagepb.Object {
-	var ret *timestamppb.Timestamp
-	if !o.RetentionExpirationTime.IsZero() {
-		ret = timestamppb.New(o.RetentionExpirationTime)
-	}
-	var ct *timestamppb.Timestamp
-	if !o.CustomTime.IsZero() {
-		ct = timestamppb.New(o.CustomTime)
+	var checksums *storagepb.ObjectChecksums
+	if len(o.MD5) > 0 {
```

Comment:
Is it possible to have both checksums?

Reply:
Yes, just checked, I for whatever reason thought it would be one or the other. Done.